resources/web/wwi/FloatingProtoParameterWindow.js

```diff
@@ -306,8 +326,11 @@ export default class FloatingProtoParameterWindow extends FloatingWindow {
     p.style.gridColumn = '2 / 2';
     p.className = 'key-parameter';
 
-    const exportCheckbox = this.#createCheckbox(parent, key);
-    p.checkbox = exportCheckbox;
+    if (this.proto.isRoot) {
+      const exportCheckbox = this.#createCheckbox(parent, key);
+      p.checkbox = exportCheckbox;
+    } else
+      p.style.marginLeft = '20px';
```

Comment:
Isn't this part already in `feature-web-proto`?

Reply:
For all others yes, but a couple were missing apparently so I assume they must be node types that have been added later on and didn't adapt them when merging it into my PR